### PR TITLE
Mobile UI Improvement for Cards Below Post Page

### PR DESCRIPTION
**Pull request by teodorus-nathaniel**
# Problem
In mobile, the current one have avatar and content side by side which makes things so cramped.
image

It also makes long texts overflows the card.
image

# Fix
The fix is to move the avatar to top
image
image

# Resolves clickup task
https://app.clickup.com/t/85zrkpuyy

### CODE REVIEW

**Comment by ical10** [comment]
All good, just a little bit weird having a small left padding for bullet points (but I guess it's by design from markdown?)

File: src/components/profiles/address-views/AuthorCard.tsx [hunk 1]
```diff
@@ -53,21 +53,20 @@ export default function AuthorCard({
...
-            >
-              <EditOutlined /> Edit
-            </ButtonLink>
-          )}
-          <FollowAccountButton address={address} />
-        </div>
-      }
+      buttons={[
+        withTipButton && !isMy && <Donate key='donate' recipientAddress={address.toString()} />,
+        isMy && profile && (
```
Comment (ical10) [inline src/components/profiles/address-views/AuthorCard.tsx:58]: Not trying to be a grammar nazi here, but at a glance it's hard to understand what does `isMy` mean. What does it refer to? Would my `isMyAddress` be a more suitable name?

**Review by ical10 — CHANGES_REQUESTED** [review]

**Comment by teodorus-nathaniel** [comment]
> All good, just a little bit weird having a small left padding for bullet points (but I guess it's by design from markdown?)

yes its from the markdown part

File: src/components/profiles/address-views/AuthorCard.tsx (hunk 1 above)
Comment (teodorus-nathaniel) [reply src/components/profiles/address-views/AuthorCard.tsx]: Renamed

**Review by ical10 — APPROVED** [review]

**Pull request merged**